src/samplers/categorical.jl

```diff
@@ -25,10 +25,16 @@ end
 
 ##### Alias Table #####
 
-struct AliasTable <: Sampleable{Univariate,Discrete}
-    accept::Vector{Float64}
-    alias::Vector{Int}
-    isampler::RangeGeneratorInt{Int,UInt}
+if Base.VERSION < v"0.7"
+    struct AliasTable <: Sampleable{Univariate,Discrete}
+        accept::Vector{Float64}
+        alias::Vector{Int}
+        isampler::RangeGeneratorInt{Int,UInt}
+else
+    struct AliasTable <: Sampleable{Univariate,Discrete}
+        accept::Vector{Float64}
+        alias::Vector{Int}
+        isampler::SamplerRangeInt{Int,UInt}
```

Comment:
Both of these blocks will need to be terminated with `end`

Reply:
Thanks.  Still doesn't compile in v0.7 since it still wants to know the definition of RangeGeneratorInt{Int,UInt} in v0.7, which makes sense.  Clearly have to call it a day.